frontend/packages/dev-console/src/components/hpa/HPAPage.tsx

```diff
@@ -35,7 +37,9 @@ const HPAPage: React.FC<PageComponentProps> = (props) => {
   const error = hpaError || workloadError?.message;
 
   const validSupportedType = VALID_HPA_TARGET_KINDS.includes(kind);
-  const title = `${hpa ? 'Edit' : 'Add'} ${HorizontalPodAutoscalerModel.label}`;
+  const title = `${hpa ? t('devconsole~Edit') : t('devconsole~Add')} ${
+    HorizontalPodAutoscalerModel.label
+  }`;
```

Comment:
it looks like this
![hpa](https://user-images.githubusercontent.com/22490998/97007265-c881f000-155e-11eb-92df-8acd4c441364.png)

try translating the whole title

Reply:
The title is rendering through variable `HorizontalPodAutoscalerModel.label` even if we add translate it will not translate the resource name.